fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/BrowserRobot.kt

```diff
@@ -404,26 +415,36 @@ class BrowserRobot {
 
     fun verifySelectAddressButtonExists(exists: Boolean) = assertUIObjectExists(selectAddressButton(), exists = exists)
 
-    fun changeCreditCardExpiryDate(expiryDate: String) =
+    fun changeCreditCardExpiryDate(expiryDate: String) {
         itemWithResId("expiryMonthAndYear").setText(expiryDate)
+        Log.i(TAG, "changeCreditCardExpiryDate: Credit card expiry date was set to: $expiryDate")
+    }
 
     fun clickCreditCardNumberTextBox() {
         mDevice.wait(Until.findObject(By.res("cardNumber")), waitingTime)
+        Log.i(TAG, "clickCreditCardNumberTextBox: Waited for credit card number text box")
```

Comment:
Maybe this would be better before the waiting starts.

Reply:
Done !